devops/Makefile

```diff
@@ -179,7 +179,7 @@ print-status:
 	@echo " +---------------------------------------------------------+ "
 
 init-project:
-	@echo "Run initial set-up for the project, including network security\n" 
```

Comment:
What is the reason behind formatting changes here?

Reply:
Comparing the files it seems that there are spaces before the carriage return at some end of lines: 
![image](https://github.com/bcgov/SIMS/assets/78114138/6b20412b-0bc5-43b9-a58d-1eff82c3f2b5)
![image](https://github.com/bcgov/SIMS/assets/78114138/85f5bc17-83da-426a-af27-1b6015092e7e)
![image](https://github.com/bcgov/SIMS/assets/78114138/c6979bd1-02d7-43ee-ba03-ad709891ab15)
![image](https://github.com/bcgov/SIMS/assets/78114138/af84856f-acbf-4459-ad91-60b0b2a114be)

Reply:
Checking the file in my local and saving the PR-changed file did not cause additional changes.
It seems that the PR file has the right formatting.